ci/Dockerfile

```diff
@@ -1,17 +1,26 @@
-# This file was inspired by envoy Dockerfile:
-# https://github.com/envoyproxy/envoy/blob/445a67344ffda0c8828c8e438e463fcaa7878434/ci/Dockerfile-envoy-alpine
-
-FROM frolvlad/alpine-glibc:alpine-3.17_glibc-2.34
+# This file was inspired by a combination of the following:
+# Istio and platform's mesh derivative approach
+# Uses upstream envoy's ubuntu versioning
+# Consuemd by edge to buld gateway-proxy and gloo pods
+# May be used as is or as a builder intermediate (e.g. for distroless)
+# ON_MINOR_UPDATE: Pull from https://github.com/envoyproxy/envoy/blob/4d46da0bba54dfb849d8bf68b600e53d87310a1a/ci/Dockerfile-envoy#L1-L2
+FROM ubuntu:focal
 
 ENV loglevel=info
 
-RUN apk upgrade --update-cache \
-    && apk add dumb-init ca-certificates \
-    && rm -rf /var/cache/apk/*
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
I missed this on the first pass -- can we get away with removing the various `-y`s in the first `RUN` command if we have this set?

Reply:
Mimicked from mesh. We may be able to but I was afraid of chestertons fence.

Reply:
Resolving per your research shared in slack: https://unix.stackexchange.com/a/701288
https://solo-io-corp.slack.com/archives/C011H3TU17T/p1696270549377469?thread_ts=1696259513.309839&cid=C011H3TU17T

Reply:
I don't know how this interacts with apt (see stackoverflow post here: https://unix.stackexchange.com/a/701288) so I say we leave it